
# HG changeset patch
# User Hans de Goede <hdegoede@redhat.com>
# Date 1264534526 -3600
# Node ID 17502820c67bea1f869cde31900ad6dd48d03693
# Parent  ea34d302fee03e23147053e0124934e08c0b3fb4
libv4l: make autogain_adjust void

From: Hans de Goede <hdegoede@redhat.com>

libv4l: make autogain_adjust void

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/processing/autogain.c	Wed Jan 20 22:58:28 2010 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/processing/autogain.c	Tue Jan 26 20:35:26 2010 +0100
@@ -40,7 +40,7 @@
 }
 
 /* Adjust ctrl value with steps steps, while not crossing limit */
-static int autogain_adjust(struct v4l2_queryctrl *ctrl, int *value,
+static void autogain_adjust(struct v4l2_queryctrl *ctrl, int *value,
   int steps, int limit)
 {
   int ctrl_range = (ctrl->maximum - ctrl->minimum) / ctrl->step;

